src/hera/input.py

```diff
@@ -48,3 +54,17 @@ def sleep(seconds: int, sentence: str = 'Hello, world!'):
 
     name: str
     parameters: List[str]
+
+
+class InputParameterAsEnv(BaseModel):
+    name: str
+    value: str
+
+    def get_argument_parameter(self):
+        return IoArgoprojWorkflowV1alpha1Parameter(name=self.name, value=self.value)
+
+    def get_input_parameter(self):
+        return IoArgoprojWorkflowV1alpha1Inputs(name=self.name)
+
+    def get_env_spec(self):
+        return EnvSpec(name=self.name, value=f"{{{{inputs.parameters.{self.name}}}}}")
```

Comment:
I love this concept. What do you think about calling this `Variable`? 🙂 The docstring can outline that the variable will be presented to the task in the form of an environment variable named `name`.